### PR TITLE
Add NfSpy: NFS UID/GID spoofer

### PR DESCRIPTION
**Pull request by Couskito**
# Description

Add NfsPy : UID/GID spoofer tool to exploit NFSv3 shares : https://github.com/bonsaiviking/NfSpy

### CODE REVIEW

File: sources/install.sh [hunk 1]
```diff
@@ -2340,6 +2340,20 @@ function install_ligolo-ng() {
   add-test-command "ligolo-proxy --help"
 }
 
+function install_nfspy() {
+  colorecho "Installing nfspy"
+  git -C /opt/tools/ clone https://github.com/bonsaiviking/NfSpy.git
+  cd /opt/tools/NfSpy
+  apt install -y libfuse-dev
```
Comment (ShutdownRepo) [inline sources/install.sh:2347]: this should be changed to `fapt libfuse-dev`

File: sources/install.sh [hunk 2]
```diff
@@ -2340,6 +2340,20 @@ function install_ligolo-ng() {
...
+  python2 -m pip install fuse-python
+  python2 setup.py install
+  chmod +x scripts/*
+  ln -v -s /opt/tools/NfSpy/scripts/nfspy /opt/tools/bin/nfspy
+  ln -v -s /opt/tools/NfSpy/scripts/nfspysh /opt/tools/bin/nfspysh
+  add-test-command "nfspy --help"
+  add-test-command "nfspysh --help"
+}
```
Comment (ShutdownRepo) [inline sources/install.sh:2355]: Can you add `add-history` if you have some command example in mind?

**Review by ShutdownRepo — CHANGES_REQUESTED** [review]

**Comment by ShutdownRepo** [comment]
Thank you for this PR. Changing target branch to `dev.

File: sources/install.sh [hunk 3]
```diff
@@ -2340,6 +2340,20 @@ function install_ligolo-ng() {
   add-test-command "ligolo-proxy --help"
 }
 
+function install_nfspy() {
+  colorecho "Installing nfspy"
+  git -C /opt/tools/ clone https://github.com/bonsaiviking/NfSpy.git
+  cd /opt/tools/NfSpy
+  apt install -y libfuse-dev
+  python2 -m pip install fuse-python
```
Comment (QU35T-code) [inline sources/install.sh:2348]: In order to work in an isolated environment, can you add a python venv like this ?

```bash
virtualenv -p /usr/bin/python2 ./venv
./venv/bin/python2 -m pip install fuse-python
./venv/bin/python2 setup.py install
chmod +x scripts/*
sed -i "s#/usr/bin/env python#/opt/tools/NfSpy/venv/bin/python2#" /opt/tools/NfSpy/scripts/nfspy
sed -i "s#/usr/bin/env python#/opt/tools/NfSpy/venv/bin/python2#" /opt/tools/NfSpy/scripts/nfspysh
ln -v -s /opt/tools/NfSpy/scripts/nfspy /opt/tools/bin/nfspy
ln -v -s /opt/tools/NfSpy/scripts/nfspysh /opt/tools/bin/nfspysh
```

**Pull request closed without merging**